### PR TITLE
wrote content for how to write zkapp page without screenshot

### PR DESCRIPTION
**Pull request by iregina**
For our non-developer users: basic instructions to learn how to use a zkApp and set the expectation that they must have a wallet filled with MINA ready to go
- Draft can be found on [Google Docs](https://docs.google.com/document/d/1ttCf0aHJIcw9h6rwOLdl1KTKE_Wh0qowcm71cCju7tE/edit?usp=sharing)
- Screenshot will be added later

Topic [preview](https://docs2-git-howtousezkapp-minadocs.vercel.app/using-mina/how-to-use-zkapp)

### CODE REVIEW

**Review by barriebyron — COMMENTED** [review]
@iregina I like this topic to help our non-developer users get up to speed quickly. Well done. 

wdyt about adding this general "typical non-developer user" context to the topic? 

I have so much to learn, I know.

File: docs/using-mina/how-to-use-zkapp.mdx [hunk 1]
```diff
@@ -2,4 +2,39 @@
...
 ---
 
-Describe
+import Subhead from '@site/src/components/common/Subhead';
+
+# Install a Wallet
+
+<Subhead>Learn how to interact with a zero-knowledge smart contract</Subhead>
+
+:::info
```
Comment (jasongitmail) [inline docs/using-mina/how-to-use-zkapp.mdx:11]: Should appear above the H1, for consistency with other pages.

File: docs/using-mina/how-to-use-zkapp.mdx [hunk 2]
```diff
@@ -2,4 +2,39 @@
...
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+### Prerequisites
+
+To use a zkApp:
+1. Install a Chrome zkApp-compatible Mina wallet.
```
[the remark this answers is not in the document]
Comment (jasongitmail) [reply docs/using-mina/how-to-use-zkapp.mdx:20]: +1 for removing `Chrome` here

File: docs/using-mina/how-to-use-zkapp.mdx [hunk 3]
```diff
@@ -2,4 +2,39 @@
...
+
+:::note
+
+The Mina community has created a variety of different wallets. Only the [Auro Wallet for Chrome](https://www.aurowallet.com) supports interactions with zkApps.
+
+:::
+
+### Instructions
+
+1. Visit the zkApp in a web browser. For example,  mycoolzkapp.com.
```
[the remark this answers is not in the document]
Comment (jasongitmail) [reply docs/using-mina/how-to-use-zkapp.mdx:31]: +1

File: docs/using-mina/how-to-use-zkapp.mdx [hunk 4]
```diff
@@ -2,4 +2,39 @@
...
+2. Interact with the zkApp as intended. For example, make a move in a game, enter in your age, and so on.
+3. Click the confirmation button to send the transaction to the Mina network.
+4. In your Mina browser wallet extension, confirm the transaction again.
```
Comment (jasongitmail) [inline docs/using-mina/how-to-use-zkapp.mdx:34]: -again 

They're only confirming it once. The button on the page starts that interactions, but they're actually confirming via the wallet.

**Review by jasongitmail — CHANGES_REQUESTED** [review]

File: docs/using-mina/how-to-use-zkapp.mdx [hunk 5]
```diff
@@ -2,4 +2,39 @@
...
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+### Prerequisites
+
+To use a zkApp:
```
Comment (jasongitmail) [inline docs/using-mina/how-to-use-zkapp.mdx:19]:
```suggestion
```
testing out the inline commit suggestion

**Comment by iregina** [comment]
> @iregina I like this topic to help our non-developer users get up to speed quickly. Well done.
> 
> wdyt about adding this general "typical non-developer user" context to the topic?
> 
> I have so much to learn, I know.

We can revisit this later - this is part of a series but open to this idea!

**Comment by iregina** [comment]
Passed Vercel with @barriebyron

**Comment by barriebyron** [comment]
@iregina  good work, all of your changes are showing. I don't have approval permissions (yet)

File: docs/using-mina/how-to-use-zkapp.mdx (hunk 1 above)
Comment (barriebyron) [reply docs/using-mina/how-to-use-zkapp.mdx:11]: @iregina  I see now... the zkApp docs have more solid examples of placement for the header

see https://github.com/o1-labs/docs2/blob/main/docs/zkapps/how-to-test-a-zkapp.mdx for a good example, move to line 6

**Review by barriebyron — CHANGES_REQUESTED** [review]

File: docs/using-mina/how-to-use-zkapp.mdx [hunk 6]
```diff
@@ -2,4 +2,38 @@
...
+:::
+
+### Prerequisites
+
+1. Install a zkApp-compatible [Mina wallet](../using-mina/install-a-wallet).
+2. Make sure that your Mina wallet contains MINA to pay for transaction fees. 
+
+:::note
+
+The Mina community has created a variety of different wallets. Only the [Auro Wallet for Chrome](https://www.aurowallet.com) supports interactions with zkApps.
```
Comment (jasongitmail) [inline docs/using-mina/how-to-use-zkapp.mdx:24]:
```suggestion
The Mina community has created a variety of different wallets. Only the [Auro Wallet for Chrome](https://www.aurowallet.com) supports interactions with zkApps currently.
```

File: docs/using-mina/how-to-use-zkapp.mdx [hunk 7]
```diff
@@ -2,4 +2,38 @@
...
+
+:::
+
+### Instructions
+
+1. Visit the zkApp in a web browser. For example, `mycoolzkapp.com`.
+2. Interact with the zkApp as intended. For example, make a move in a game, enter in your age, and so on.
+3. Click the confirmation button to send the transaction to the Mina network.
+4. In your Mina browser wallet extension, confirm the transaction.
+5. Done! Congratulations. Your transaction is processed by the Mina network and the zkApp state is updated.
```
Comment (jasongitmail) [inline docs/using-mina/how-to-use-zkapp.mdx:34]:
```suggestion
5. Done! Congratulations. Your transaction will be processed by the Mina network and, when accepted into a block, will update the zkApp's on-chain state.
```

Comment (jasongitmail) [reply docs/using-mina/how-to-use-zkapp.mdx:34]: We need to use future tense here b/c the transaction being sent to the network sits there until processed into a block. Only if/when it is processed into a block, will it update the zkApp account on chain.

**Review by jasongitmail — COMMENTED** [review]
@iregina   
- Can you move the zkApp banner to the top of the page, for consistency with its usage on other pages please? 
- Side bar capitalization is inconsistent with other page in that section. For the current moment, we should be consistent with those until all are updated simultaneously. Screenshot 2023-03-23 at 1 32 30 AM

**Comment by iregina** [comment]
>

**Review by barriebyron — APPROVED** [review]
lgtm!

**Comment by barriebyron** [comment]
@jasongitmail  what is the process to get your approval on this PR after you have requested changes and those changes have been made? Does @iregina  re-request or? Asking here to learn and because this PR cannot be merged with "Changes requested" review status in place

**Review by jasongitmail — APPROVED** [review]

**Comment by barriebyron** [comment]
Congratulations @iregina  on this great intro topic to help new devs use zkApps!

**Pull request merged**